src/modules/auth/stores/auth.store.ts

```diff
@@ -0,0 +1,47 @@
+import { useLocalStorage } from '@vueuse/core/index'
+import { defineStore } from 'pinia'
+import { computed, ref } from 'vue'
+
+import type { AuthLoginForm } from '@/models/auth/login/authLoginForm.model'
+import type { CurrentUser } from '@/models/user/current/currentUser.model'
+import { authService } from '@/modules/auth/services/auth.service'
+
+export const useAuthStore = defineStore('auth', () => {
+	const currentUser = ref<CurrentUser | null>(null)
+	const accessToken = useLocalStorage<string | null>('accessToken', '')
+
+	const isAuthenticated = computed<boolean>(() => currentUser.value === null)
+
+	async function getCurrentUser(): Promise<CurrentUser | null> {
+		if (currentUser.value !== null) {
+			return currentUser.value
+		}
+
+		currentUser.value = await authService.getCurrentUser()
+		return currentUser.value!
+	}
+
+	function setCurrentUser(user: CurrentUser | null): void {
+		currentUser.value = user
+	}
+
+	async function login(data: AuthLoginForm): Promise<void> {
+		const response = await authService.login(data.username!, data.password!)
```

Comment:
waarom die ! achter de variablen ?

Reply:
Webstorm wou, voor de herinstalatie, niet compilen wanneer ik dit niet deed. Alles is nu weer in orde!